### ISSUE TITLE
OLS not showing InverseOf axioms

### ISSUE THREAD

**Issue by cmungall** [EBISPOT/OLS#146]
Following from https://github.com/pato-ontology/pato/issues/112#issuecomment-306049653

E.g.
http://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FBFO_0000051

should show 'part of'

**Comment by simonjupp** [comment]
Yep, this isn't implemented yet, but we will add all property axioms and characteristics soon. 
Thanks
